README.md

```diff
@@ -232,21 +242,46 @@ my-s3-bucket:
 
 ## Postgres database
 
-A postgres database can be added by configuring the `extensions.yml` file. Below is a simple example configuration, showing some of the available options:
+A postgres database can be added by configuring the `platform-config.yml` file. Below is a simple example configuration, showing some of the available options:
 
 ```yaml
 my-postgres-db:
   type: postgres
   version: 16.2
   environments:
-    "*":
+    '*':
       plan: tiny
       backup_retention_days: 1 # Optional.  Must be between 1 and 35.  If none, defaults to 7.
     prod:
       deletion_protection: true # Optional
       deletion_policy: Retain # Optional: Delete or Retain
 ```
 
+## S3 to S3 data migration module
+
+This module will create cross account permissions to write to your S3 bucket, allowing the copying of files from a source S3 bucket to your destination S3 bucket.  Any cross account data migration **must be approved by cyber**.  Please see the [S3 to S3 data migration documentation](https://platform.readme.trade.gov.uk/reference/terminology/) for further details on using this module.
+
+Most users will not have permissions to apply the following configuration.
+In this case, an SRE team member or someone from the DBT-platform team will be able to help once cyber has approved the request.
+
+S3 data migration can be enabled by adding the `data_migration` parameter along with the `import` parameter and its mandatory configuration to the `<application>-s3-bucket` extension in your `platform-config.yml` file. The `source_kms_key_arn` is optional as it depends on whether the source bucket has KMS key encryption on it.
```

Comment:
I don't think we shou.d be implying a naming convention here, it's up to the teams...
```suggestion
S3 data migration can be enabled by adding the `data_migration` parameter along with the `import` parameter and its mandatory configuration to the S3 extension in your `platform-config.yml` file. The `source_kms_key_arn` is optional as it depends on whether the source bucket has KMS key encryption on it.
```